crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -9,10 +9,10 @@
   <header class="page-header-background">
     <div class="content-wrapper">
       <h1 class="padding-top-5 margin-0">
-        Contact the Department of Justice about a civil rights concern
+        Submit a civil rights complaint to the Department of Justice
```

Comment:
@ilodidoj, based on our team discussion, we think this should read: "Submit a civil rights concern to the Department of Justice"

Reply:
See comment here: https://docs.google.com/document/d/1bxSRc9CQOHjQhTaYkHncE4tykUEFeC6RzFtP5IQk0Ic/edit#.